docs/posts/ibis-version-v8-release/index.qmd

```diff
@@ -0,0 +1,238 @@
+---
+title: "Ibis v8: streaming and more!"
+author: "Ibis team"
+date: "2024-02-05"
+categories:
+    - release
+    - blog
+---
+
+## Overview
+
+Ibis v8 marks the first release of data streaming backends in Ibis! We're
```

Comment:
`v8` vs. `8.0`?
are we going for the tomato juice market or the webassembly market?

Reply:
I prefer `v8` but might be missing some nuance there